internal/services/icinga2/docker.go

```diff
@@ -171,6 +171,20 @@ func (n *dockerInstance) WriteConfig(file string, data []byte) {
 	}
 }
 
+func (n *dockerInstance) DeleteConfigGlob(glob string) {
+	logger := n.logger.With(zap.String("glob", glob))
+
+	stderr := utils.NewLineWriter(func(line []byte) {
+		logger.Error("error from container while deleting file", zap.ByteString("line", line))
+	})
+
+	err := utils.DockerExec(context.Background(), n.icinga2Docker.dockerClient, n.logger, n.containerId,
+		[]string{"perl", "-e", "unlink glob @ARGV[0]", "--", "/" + glob}, nil, nil, stderr)
```

Comment:
Why the implicit slash?

Reply:
To be consistent with `WriteConfig`: https://github.com/Icinga/icinga-testing/blob/68d4825f41cd8ed8a46d33505a480ab832e127ae/services/icinga2.go#L26-L31

(Reason there is that the original thought was to not be too docker-specific and for example also allow running multiple `icinga2` processes locally with lots of `-D` parameters. But I think that idea is gone anyway, so it's just consistency now.)